SharedSource/Development/RhinoMocks/UnitTesting/Threading/LockingDecoratorTestHelper.cs

```diff
@@ -46,19 +47,21 @@ public LockingDecoratorTestHelper (T lockingDecorator, object lockObject, T inne
     }
 
     public void ExpectSynchronizedDelegation<TResult> (Func<T, TResult> action, TResult fakeResult)
+        where TResult : notnull
     {
       ArgumentUtility.CheckNotNull ("action", action);
       ArgumentUtility.CheckNotNull ("fakeResult", fakeResult);
 
       ExpectSynchronizedDelegation (action, fakeResult, r => Assert.That (r, Is.EqualTo (fakeResult)));
     }
 
-    public void ExpectSynchronizedDelegation<TResult> (Func<T, TResult> action, TResult fakeResult, Action<TResult> resultChecker)
+    public void ExpectSynchronizedDelegation<TResult> (Func<T, TResult> action, TResult fakeResult, Action<TResult> resultChecker) where TResult: notnull

```

Comment:
Formatting.

```suggestion
    public void ExpectSynchronizedDelegation<TResult> (Func<T, TResult> action, TResult fakeResult, Action<TResult> resultChecker) where TResult : notnull
```

Multiple occurrences.